ras/sosreport.py

```diff
@@ -57,14 +57,14 @@ def setUp(self):
             self.sos_cmd = "sosreport"
         elif dist.name in ['rhel', 'centos', 'fedora']:
             sos_pkg = 'sos'
+            self.sos_cmd = "sos report"
         else:
             self.cancel("sosreport is not supported on %s" % dist.name)
         for package in (sos_pkg, 'java'):
             if not sm.check_installed(package) and not sm.install(package):
                 self.cancel(
                     "Package %s is missing and could not be installed" % (package))
         if dist.name == "rhel":
-            self.sos_cmd = "sos report"
```

Comment:
Can you also move this dist.name == rhel code block to the elif block? This will make the code readable and compact. At present the sos vs sosreport logic is split into 2 code blocks.

Reply:
I have updated the code logic inside the elif block as said 
Thank you.